static/app/views/organizationIntegrations/addIntegrationButton.tsx

```diff
@@ -1,23 +1,24 @@
 import * as React from 'react';
 
-import Button from 'sentry/components/button';
+import Button, {ButtonProps} from 'sentry/components/button';
 import Tooltip from 'sentry/components/tooltip';
 import {t} from 'sentry/locale';
 import {IntegrationWithConfig} from 'sentry/types';
 
 import AddIntegration from './addIntegration';
 
-type Props = {
+interface AddIntegrationButtonProps
+  extends Omit<ButtonProps, 'aria-label'>,
```

Comment:
should we use `ButtonPropsWithoutAriaLabel` here?

Reply:
Hmm, actually, the only reason it's omitted is because we have a value for aria label which makes it misleading (see a few lines down, we pass the aria label). I'll change it

Reply:
Sorry this comment was actually meant for the [static/app/components/globalModal/components.tsx](https://github.com/getsentry/sentry/pull/31699/files/b0385db56bcf8e805f358f4d4f5cec2e388430db#diff-705707b3040b387c0a059c46113c348c20c7e672939cd51f11f5d5e6114af88d) file